test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m

```diff
@@ -134,3 +139,78 @@
 
 % change the directory
 cd(currentDir)
+
+% test getMappingInfo
+[reac,micRea,binOrg,patOrg,reacPat,reacNumb,reacSet,reacTab,reacAbun,reacNumber]=getMappingInfo(models,abunFilePath,indNumb);
+
+assert(exist('reac', 'var') == 1)
+assert(exist('micRea', 'var') == 1)
+assert(exist('binOrg', 'var') == 1)
+assert(exist('patOrg', 'var') == 1)
+assert(exist('reacPat', 'var') == 1)
+assert(exist('reacNumb', 'var') == 1)
+assert(exist('reacSet', 'var') == 1)
+assert(exist('reacTab', 'var') == 1)
+assert(exist('reacAbun', 'var') == 1)
+assert(exist('reacNumber', 'var') == 1)
+
+% test checkNomenConsist
+[autoStat,fixVec,organisms]=checkNomenConsist(organisms,autoFix);
+
+assert(exist('autoStat', 'var') == 1)
+assert(exist('fixVec', 'var') == 1)
+assert(length(organisms) == 5)
+
+% test fastSetupCreator
+
+setup=fastSetupCreator(models, organisms, {},objre);
+ 
+assert(exist('setup', 'var') == 1)
+assert(strcmp(class(setup),'struct') == 1)
+assert((length(setup.S) == length(setup.rxns)) == 1)
```

Comment:
As further down:
could you please use `size()` for matrices?
Also:
Could `verifyCobraModel` make sense here?